.luacov

```diff
@@ -20,12 +20,20 @@ return {
         -- (happens with integration tests)
         "^lib/",
         "^awesomerc.lua",
+
+        -- Include awesomerc.lua (in build dir with integration tests).
+        escape_pattern(build) .. "/awesomerc.lua",
```

Comment:
This might not be required when using a relative path for the RC file (awesome gets run from the build dir already)?!